server/request_processor.go

```diff
@@ -166,17 +167,17 @@ func (r *RpcRequest) sendTxToRelay() {
 
 	// Check if tx was already forwarded and should be blocked now
 	if r.blockResendingTxToRelay(txHash) {
-		r.logger.log("[sendTxToRelay] blocked %s", txHash)
+		r.logger.Info("[sendTxToRelay] Blocked", "tx", txHash)
 		r.writeRpcResult(txHash)
 		return
 	}
 
-	r.logger.log("[sendTxToRelay] sending %s -- from ip: %s / address: %s / to: %s", txHash, r.ip, r.txFrom, r.tx.To())
+	r.logger.Info(fmt.Sprintf("[sendTxToRelay] sending %s -- from ip: %s / address: %s / to: %s", txHash, r.ip, r.txFrom, r.tx.To()))
```

Comment:
please log variables separately here (no `fmt.Sprintf`) - this makes it easier to query the logs

Reply:
updated